L1Trigger/L1TMuonBarrel/src/L1TMuonBarrelParamsHelper.cc

```diff
@@ -36,7 +36,7 @@ void L1TMuonBarrelParamsHelper::configFromPy(std::map<std::string, int>& allInts
 	setAssLUTPath(AssLUTpath);
 	///Read Pt assignment Luts
 	std::vector<LUT> pta_lut(0); pta_lut.reserve(19);
-	std::vector<int> pta_threshold(6); pta_threshold.reserve(9);
+	std::vector<int> pta_threshold(6); pta_threshold.reserve(10);
```

Comment:
@davidlange6 No, I meant reserve, as it was originally used by the developer, as the intended estimate of the capacity of the vector is known = 10.
This would only allocate memory, but leave vector elements uninitialized.  They are filled
later on in the loop.  But I guess either one reserve() and resize() would work.

In any case, there were no more "Invalid writes" from BMTF as originally reported by valgrind.